platforms/java/com/area9innovation/flow/Native.java

```diff
@@ -1137,7 +1137,29 @@ public static final boolean isStructName(String name) {
 		return FlowRuntime.struct_ids.containsKey(name);
 	}
 
-	public static final int extractFuncArity(Object fn) {
```

Comment:
Why is it a good idea to have this magic with strings?

Reply:
The main purpose of this PR is to prepare runtime for dynamically calling functions by their name as string. It may be used in `eval` function. Now `eval` interprets all calls in AST down to natives, which are called, but with calls by name as string enabled (i.e. with `hostCall`), `eval` may use many of functions from runtime directly.